.circleci/config.yml

```diff
@@ -773,7 +773,7 @@ jobs:
       - run: mkdir -p $TEST_RESULTS
 
       - run-acceptance-tests:
-          additional-flags: -kubeconfig="$primary_kubeconfig" -secondary-kubeconfig="$secondary_kubeconfig" -enable-pod-security-policies -enable-transparent-proxy -consul-image=hashicorppreview/consul-enterprise:1.15-dev-23aaa4f83845d0e2eced9ea69f731d7eedf840d1
```

Comment:
Does -pod-security-policies break things on the new versions?

Reply:
Kubernetes no longer supports pod security policies starting 1.25. https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/